### PR TITLE
chore: Enable unit-tests in CI

### PR DESCRIPTION
**Pull request by chakaz**

### CODE REVIEW

**Review by romange — APPROVED** [review]

**Comment by kostasrim** [comment]
only one failing :)

**Comment by chakaz** [comment]
Actually there are 2 failing, one flaky and one constantly :(

File: src/facade/facade_test.cc [hunk 1]
```diff
@@ -6,19 +6,34 @@
...
 #include <absl/strings/match.h>
 
+#include "base/logging.h"
+
 namespace facade {
 
 using namespace testing;
 using namespace std;
 
 bool RespMatcher::MatchAndExplain(const RespExpr& e, MatchResultListener* listener) const {
```
Comment (adiholden) [inline src/facade/facade_test.cc:16]: do we have to use const & for const RespExpr& e ?
if not we can skip the copy of line 17

Comment (chakaz) [reply src/facade/facade_test.cc]: Good idea! Done :)

**Review by adiholden — APPROVED** [review]

**Comment by chakaz** [comment]
That was exhausting! and we still have some flaky tests :no_mouth:

**Pull request merged**